### DISCUSSION TITLE
总览

### DISCUSSION THREAD

**Discussion by giscus** [oldwinter/knowledge-garden#22] [主发布站的评论系统]
# 总览

我的第二大脑 second brain，我的数字花园 digital garden，用obsidian双链笔记软件写作而成。

https://ob.tianzhongs.ml/

**Comment by songshua295** [comment]
评论测试

**Reply by oldwinter** [reply]
哥们，你网站下面没有评论，怎么能评论到我这来的，是已经移除了嘛。如果需要可以自己去giscus生成那段script代码哦。

**Comment by songshua295** [comment]
是的，我测试部署了一下，发现有一些问题，所以移除了&nbsp;


tianzhongs
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "DongDong ***@***.***&gt;; 
发送时间: 2022年9月14日(星期三) 中午1:30
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [oldwinter/knowledge-garden] 总览 (Discussion #22)





 
哥们，你网站下面没有评论，怎么能评论到我这来的，是已经移除了嘛。如果需要可以自己去giscus生成那段script代码哦。
